src/actor.js

```diff
@@ -259,3 +261,68 @@ export const readyFreddy = () => {
         console.log(`WARNING: ${ENV_VARS.WATCH_FILE} environment variable not specified, readyFreddy() has no effect.`);
     }
 };
+
+export const call = (actId, opts = {}) => {
+    const { acts, keyValueStores } = apifyClient;
+
+    // Common options.
+    const { token } = opts;
+    checkParamOrThrow(actId, 'actId', 'String');
+    checkParamOrThrow(token, 'token', 'Maybe String');
+    const defaultOpts = { actId };
+    if (token) defaultOpts.token = token;
+
+    // RunAct() options.
+    const { build, body, contentType } = opts;
+    checkParamOrThrow(build, 'build', 'Maybe String');
+    checkParamOrThrow(body, 'body', 'Maybe Buffer | String');
+    checkParamOrThrow(contentType, 'contentType', 'Maybe String');
+    const runActOpts = {};
+    if (contentType) runActOpts.contentType = contentType;
+    if (build) runActOpts.build = build;
+    if (body) runActOpts.build = body;
+
+    // GetAct() options.
+    const { timeoutSecs = DEFAULT_APIFY_CALL_WAIT_SECS, fetchOutput = true } = opts;
```

Comment:
I think by default there should be no timeout, otherwise it might surprise users